client/index.html

```diff
@@ -13,40 +21,51 @@
-    <script src="/app/station/stations.controller.js"></script>
-    <script src="/app/station/station.detail.controller.js"></script>
-    <script src="/app/station/stations.service.js"></script>
-    <script src="/app/profile/profile.js"></script>
-    <script src="/app/profile/profile.controller.js"></script>
-    <script src="/app/playlist/playlist.js"></script>
-    <script src="/app/playlist/playlist.service.js"></script>
-    <script src="/app/playlist/playlist.controller.js"></script>
-    <script src="/app/playlist/userplaylist.service.js"></script>
-    <script src="/app/playlist/playlist.detail.controller.js"></script>
-    <script src="/app/components/player/player.controller.js"></script>
+    <!-- build:js(client) js/vendor.min.js -->
+    <!-- bower:js -->
+    <script src="lib/jquery/dist/jquery.js"></script>
+    <script src="lib/angular/angular.js"></script>
+    <script src="lib/angular-route/angular-route.js"></script>
+    <script src="lib/bootstrap/dist/js/bootstrap.js"></script>
+    <!-- endbower -->
+    <!-- endbuild -->
+    <!-- build:js({client,.tmp}) js/bundle.min.js -->
+    <!-- inject:js -->
+    <script src="components/search/search.js"></script>
+    <script src="components/search/search.service.js"></script>
+    <script src="components/player/player.controller.js"></script>
+    <script src="components/nav/nav.js"></script>
+    <script src="components/nav/nav.directive.js"></script>
+    <script src="components/nav/nav.controller.js"></script>
+    <script src="app/account/auth.js"></script>
+    <script src="components/auth/user.service.js"></script>
+    <script src="components/auth/auth.service.js"></script>
+    <script src="components/auth/auth.interceptor.service.js"></script>
+    <script src="app/station/stations.js"></script>
+    <script src="app/station/stations.service.js"></script>
+    <script src="app/station/stations.controller.js"></script>
+    <script src="app/station/station.detail.controller.js"></script>
+    <script src="app/result/result.js"></script>
+    <script src="app/result/result.controller.js"></script>
+    <script src="app/profile/profile.js"></script>
+    <script src="app/profile/profile.controller.js"></script>
+    <script src="app/podcast/podcast.js"></script>
+    <script src="app/podcast/podcast.service.js"></script>
+    <script src="app/podcast/podcast.controller.js"></script>
+    <script src="app/playlist/userplaylist.service.js"></script>
+    <script src="app/playlist/playlist.js"></script>
+    <script src="app/playlist/playlist.service.js"></script>
+    <script src="app/playlist/playlist.detail.controller.js"></script>
+    <script src="app/playlist/playlist.controller.js"></script>
+    <script src="app/home/popular.js"></script>
+    <script src="app/home/popular.controller.js"></script>
+    <script src="app/account/auth.controller.js"></script>
+    <script src="app/constants.js"></script>
+    <script src="app/app.js"></script>
+    <!-- endinject -->
```

Comment:
Ya. For development, the gulp-inject tasks will find the files based on the gulp.src and inject the script tags. Then for deployment, gulp-useref will look at those injected tags and concat and uglify/minify.